documentation/release-process.md

```diff
@@ -0,0 +1,56 @@
+# Release Process Documentation
+
+Based on https://github.com/dotnet/sdk/issues/41607
+
+## Ideal Process
+
+**Note:** This process may vary based on the workload owners.
+**Note:** Most of this process today is done through disparate builds in each workload repo and this repo (or manually). Below is outlining our desired goal.
+
+1. **Publishing Release-Ready Versions**
+    - Each workload owner publishes release-ready versions to the `dotnet8-workloads` and `dotnet9-workloads` channels.
+
+2. **Creating Workload Set Repo PRs**
+    - Workload set repository pull requests (PRs) are created based on the release channel.
+
+3. **Review and Merge PRs**
+    - A person reviews the PR, approves it, merges it, and a build is triggered automatically.
+
+5. **Build Process**
+    - There are a few phases to the build process:
+        1. Create a stable and unstable version of the workload set (a stable version is only needed in servicing).
+            - 9.0.102-servicing.12345.6
+            - 9.0.102
+        2. Publish the unstable version to the workloads feed.
+        3a. If we're not in servicing/golive:
+            - Publish runtime, emsdk, maui, and aspire to the workloads feed.
+        3b. If we're in servicing/golive:
+            - Only publish maui and the unstable workload set to the workloads feed.
+            - Additionally, create a stable workloads feed and publish the runtime, emsdk, maui, aspire, and stable workload to that feed.
+              - To enable this step, the .NET staging pipeline iwll have to be modified to publish the runtime and emsdk builds to the appropriate workloads channels. Today, that publishing is done in the runtime public build.
```

Comment:
Markdown doesn't have an `3a`/`3b` concept. It might work well as:

```suggestion
        3. The process is different depending on if we're in servicing/golive.
            1. If we're **not** in servicing/golive:
                - Publish runtime, emsdk, maui, and aspire to the workloads feed.
            2. If we're in servicing/golive:
                - Only publish maui and the unstable workload set to the workloads feed.
                - Additionally, create a stable workloads feed and publish the runtime, emsdk, maui, aspire, and stable workload to that feed.
                  - To enable this step, the .NET staging pipeline will have to be modified to publish the runtime and emsdk builds to the appropriate workloads channels. Today, that publishing is done in the runtime public build.
```

Just tested this and it turns the second ordered list into `i.` and `ii.` when rendered. Also, my suggestion fixed a typo with `iwll` to `will`.